Sources/X509/Extensions.swift

```diff
@@ -89,6 +89,11 @@ extension Certificate {
         public init<Elements>(_ extensions: Elements) throws where Elements: Sequence, Elements.Element == Extension {
             self._extensions = Array(extensions)
             
+            let maxExtensions = 16
```

Comment:
I think we can safely raise this to 32 without hurting us at all, and it's probably safer.